diff --git a/node_modules/@ckeditor/ckeditor5-paste-from-office/src/filters/space.js b/node_modules/@ckeditor/ckeditor5-paste-from-office/src/filters/space.js
index a2f63e4..3170b2d 100644
--- a/node_modules/@ckeditor/ckeditor5-paste-from-office/src/filters/space.js
+++ b/node_modules/@ckeditor/ckeditor5-paste-from-office/src/filters/space.js
@@ -36,9 +36,16 @@ export function normalizeSpacing(htmlString) {
  */
 export function normalizeSpacerunSpans(htmlDocument) {
     htmlDocument.querySelectorAll('span[style*=spacerun]').forEach(el => {
-        const htmlElement = el;
-        const innerTextLength = htmlElement.innerText.length || 0;
-        htmlElement.innerText = Array(innerTextLength + 1).join('\u00A0 ').substr(0, innerTextLength);
+        // const htmlElement = el;
+        // const innerTextLength = htmlElement.innerText.length || 0;
+        // htmlElement.innerText = Array(innerTextLength + 1).join('\u00A0 ').substr(0, innerTextLength);
+        // https://github.com/ckeditor/ckeditor5/pull/10471/files
+        // For paste contents from WPS Office, the span's inner text is not all of spaces or blank,
+		    // so we should keep the origin contents, or replace ONLY space characters.
+        if ( /[^\b]/.test( el.innerText.trim() ) === false ) {
+          const innerTextLength = el.innerText.length || 0;
+          el.innerHTML = Array( innerTextLength + 1 ).join( '\u00A0 ' ).substr( 0, innerTextLength );
+        }
     });
 }
 /**
