diff --git a/src/filters/space.js b/src/filters/space.js
index a2f63e409de1a8b01d749de25fa8e43885b263e9..d3b9696dbe9f0af9772dd76869b09fa7c6068831 100644
--- a/src/filters/space.js
+++ b/src/filters/space.js
@@ -37,8 +37,10 @@ export function normalizeSpacing(htmlString) {
 export function normalizeSpacerunSpans(htmlDocument) {
     htmlDocument.querySelectorAll('span[style*=spacerun]').forEach(el => {
         const htmlElement = el;
-        const innerTextLength = htmlElement.innerText.length || 0;
-        htmlElement.innerText = Array(innerTextLength + 1).join('\u00A0 ').substr(0, innerTextLength);
+        if (/[^\b]/.test( el.innerText.trim() ) === false) {
+          const innerTextLength = el.innerText.length || 0;
+          htmlElement.innerText = Array(innerTextLength + 1).join('\u00A0 ').substr(0, innerTextLength); 
+        }  
     });
 }
 /**
